nodejs/week1/warmup/warmup.js

```diff
@@ -0,0 +1,35 @@
+class Circle {
+  constructor(radius) {
+    this.radius = radius;
+  }
+
+  getDiameter() {
+    let diameter = this.radius * 2
+    return diameter.toFixed(2);
+  }
+
+  getCircumference() {
+    let circumference = this.radius * 2 * Math.PI
+    return circumference.toFixed(2);
+  }
+
+  getArea() {
+    let area = this.radius * this.radius * Math.PI
+    return area.toFixed(2);
+  }
```

Comment:
Using toFixed transforms the value into a string. If someone using this function expects to use the result for math operations (which is quite likely) they are going to get an awful surprise. Because they'll try operating strings with numbers and chaos will ensue.

Try adding "10" + 5 to see what happens :)

I think that for accuracy you should just return the number as is. And let whoever module is in charge of presenting this data to a user to do the formatting at the very end (using toFixed for example). 

You may want to sort the methods alphabetically. That can make them easier to find. This is not required but you may encounter that different companies have styleguides which such recommendations.